### ISSUE TITLE
Does not display default icon if there is no Internet connection

### ISSUE THREAD

**Issue by jkugler** [johno/ember-cli-gravatar#66]
Using version 3.9.0

We're running our application in VMs on a host machine to do a demo. We're doing a conditional gravatar like so:
```
        {{#has-gravatar class="img-circle" email=profile.email as |g| }}
          {{#if g.has}}
            {{g.image}}
          {{else}}
            <img src="/avatar/default.jpg"/>
          {{/if}}
        {{/has-gravatar}}
```
If the browser accessing the app (it's running on the host running the VM) has an Internet connection, it correctly detects the user has no gravatar, and displays the default. However, if the browser has no Internet access (which it won't during the demo), there is no graphic displayed.

Why not? Shouldn't a lack of connection or name resolution failure mean that `g.has` is false, and thus we get the default `img src`?

**Comment by jkugler** [comment]
Now that #67 is merged, when might another version be cut? We're itching to use this. :)

**Comment by johno** [comment]
Running the tests locally now to sanity check, should be a matter of minutes 😸

**Comment by johno** [comment]
Just released as `ember-cli-gravatar@3.10.1`. Let me know if you encounter any issues. And thanks for the contribution!
